src/sonic-config-engine/minigraph.py

```diff
@@ -1142,7 +1142,26 @@ def parse_spine_chassis_fe(results, vni, lo_intfs, phyport_intfs, pc_intfs, pc_m
 #
 ###############################################################################
 
-def filter_acl_table_bindings(acls, neighbors, port_channels, sub_role):
+def filter_acl_table_for_backend(acls, vlan_members):
+    filter_acls = {}
+    for acl_name in acls:
+        if 'everflow' not in acl_name.lower():
+            filter_acls[acl_name] = acls[acl_name]
+
+    filter_acls['DATAACL'] = { 'policy_desc': 'DATAACL',
```

Comment:
>filter_acls [](http://example.com/codeflow?start=4&length=11)

You can assign `ports` in the same line, just move this line to end of function.

Reply:
Done